packages/core/src/2d/text/Font.ts

```diff
@@ -0,0 +1,50 @@
+import { RefObject } from "../../asset/RefObject"
+import { Engine } from "../../Engine";
+import { UpdateFlag } from "../../UpdateFlag";
+import { UpdateFlagManager } from "../../UpdateFlagManager";
+
+/**
+ * Font.
+ */
+export class Font extends RefObject {
```

Comment:
`Font` generation can come from two ways：
- System-font
- Use ResourceManager to load font file 

For system-font we can add two static API to support generation `Font`:

- static getOSInstalledFontNames():string[]; // Get names of fonts installed on the OS.
- static createFontFromOS(fontName:string):void; // Create Font from system-installed fonts.

Reply:
done